Makefile

```diff
@@ -294,6 +294,33 @@ build-install:
 	cp ./target/release/wasmer ./install/bin/
 	tar -C ./install -zcvf wasmer.tar.gz bin/wapm bin/wasmer
 
+UNAME_S := $(shell uname -s)
+
+build-capi:
+	mkdir -p ./capi/
+	mkdir -p ./capi/include
+	mkdir -p ./capi/lib
+
+ifeq ($(OS), Windows_NT)
+	cp target/release/wasmer_runtime_c_api.dll ./capi/lib/wasmer.dll
+	cp target/release/wasmer_runtime_c_api.lib ./capi/lib/wasmer.lib
+else
+ifeq ($(UNAME_S), Darwin)
+	cp target/release/libwasmer_runtime_c_api.dylib ./capi/lib/libwasmer.dylib
+	cp target/release/libwasmer_runtime_c_api.dylib ./capi/lib/libwasmer.a
+	# Fix the rpath for the dylib
+	install_name_tool -id "@rpath/libwasmer.dylib" ./capi/lib/libwasmer.dylib
+else
+	cp target/release/libwasmer_runtime_c_api.so ./capi/lib/libwasmer.so
+	cp target/release/libwasmer_runtime_c_api.a ./capi/lib/libwasmer.a
+endif
+endif
+
+	find target/release/build -name 'wasmer.h*' -exec cp {} ./capi/include ';'
+	cp LICENSE ./capi/LICENSE
+	cp lib/runtime-c-api/distribution/* ./capi/
```

Comment:
The `distribution` directory is added in this PR.
What do you think adding the `doc/` directory instead? It already contains [an `index.md` file](https://github.com/wasmerio/wasmer/blob/master/lib/runtime-c-api/doc/index.md) that can be used as the `README.md` for this kind of archive (modulo some reformulations). Thoughts?

The idea behind my proposal is to limit the number of `README.md` and documentation to maintain.

Reply:
That's a good point. I just fixed it to reuse the same `doc/index.md` file!